burnman/constants.py

```diff
@@ -19,3 +19,10 @@
 available in older versions.
 """
 Boltzmann = 1.3806488e-23 
+
+
+"""
+Newtonian constant of gravitation (G) in m^3 kg^-1 s^-2
+"""
+
+gravitational = scipy.constants.G
```

Comment:
Not sure about this name--it's an adjective for a constant value. Perhaps Big_G, or just G, since it is namespaced by constants.py?